jobfunnel/backend/scrapers/monster.py

```diff
@@ -334,3 +376,24 @@ def _convert_radius(self, radius: int) -> int:
         elif radius >= 200:
             radius = 200
         return radius
+
+
+class MonsterScraperUKEng(BaseMonsterScraper, BaseUKEngScraper):
+    """Scrapes jobs from www.monster.co.uk
+    """
+    def _convert_radius(self, radius: int) -> int:
+        """convert radius in miles TODO replace with numpy
+        """
+        if radius < 5:
+            radius = 0
+        elif 5 <= radius < 10:
+            radius = 5
+        elif 10 <= radius < 20:
+            radius = 10
+        elif 20 <= radius < 50:
+            radius = 20
+        elif 50 <= radius < 100:
+            radius = 50
+        elif radius >= 100:
+            radius = 100
+        return radius
```

Comment:
Missing newline, also this logic should be moved to a shared method between canada and uk